src/libraries/System.Net.Http/src/System/Net/Http/HttpIOException.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+
+namespace System.Net.Http
+{
+    /// <summary>
+    /// An exception thrown when an error occurs while reading the response content stream.
```

Comment:
```suggestion
    /// An exception thrown when an error occurs while reading the response.
```

Nit: "content stream" reads to me like it's referring only to the body